ow_sim_tests/test/arm_check_action.py

```diff
@@ -52,67 +54,70 @@ def test_02_guarded_move(self):
       server_timeout = 15.0
     )
 
-  def test_03_grind(self):
-    grind_result = test_arm_action(self,
-      'Grind', ow_lander.msg.GrindAction,
-      ow_lander.msg.GrindGoal(
+  def test_03_task_grind(self):
+    task_grind_result = test_arm_action(self,
+      'TaskGrind', owl_msgs.msg.TaskGrindAction,
+      owl_msgs.msg.TaskGrindGoal(
         x_start = 1.65,
         y_start = 0.0,
         depth = 0.05,
         length = 0.6,
         parallel = True,
         ground_position = DEFAULT_GROUND_HEIGHT
       ),
-      TEST_NAME, "test_03_grind"
+      TEST_NAME, "test_03_task_grind"
     )
 
-  def test_04_dig_circular(self):
-    dig_circular_result = test_arm_action(self,
-      'DigCircular', ow_lander.msg.DigCircularAction,
-      ow_lander.msg.DigCircularGoal(
-        x_start = 1.65,
-        y_start = 0.0,
-        depth = 0.01,
+  def test_04_task_scoop_circular(self):
+    task_scoop_circular_result = test_arm_action(self,
+      'TaskScoopCircular', owl_msgs.msg.TaskScoopCircularAction,
+      owl_msgs.msg.TaskScoopCircularGoal(
+        frame = 0,
+        relative =False,
+        point = Point(1.65, 0.0, ow_lander.constants.DEFAULT_GROUND_HEIGHT),
+        depth = 0.1,
         parallel = True,
-        ground_position = DEFAULT_GROUND_HEIGHT
       ),
-      TEST_NAME, 'test_04_dig_circular'
+      TEST_NAME, 'test_04_task_scoop_circular'
     )
 
-  def test_05_discard(self):
-    discard_result = test_arm_action(self,
-      'Discard', ow_lander.msg.DiscardAction,
-      ow_lander.msg.DiscardGoal(
-        discard = Point(1.5, 0.8, 0.65)
+  def test_05_task_discard_sample(self):
+    task_discard_sample_result = test_arm_action(self,
+      'TaskDiscardSample', owl_msgs.msg.TaskDiscardSampleAction,
+      owl_msgs.msg.TaskDiscardSampleGoal(
+        frame = 0,
+        relative = False,
+        point = Point(1.5, 0.8, 0.65),
+        height = 0.0,
```

Comment:
This required a little more explanation by me. `point` is the point on the terrain where sample will be discarded, and `height` is how high it will be dropped from. This happened to work fine because it still puts the scoop at a fairly high position, but I would rather have non-zero height.
```suggestion
        point = Point(1.5, 0.8, ow_lander.constants.DEFAULT_GROUND_HEIGHT),
        height = 0.7,
```